lib/tasks/sr_export.rake

```diff
@@ -69,9 +69,18 @@ def quiz_sr_api_data(quiz)
     sis_pseudonym = SisPseudonym.for(teacher, LoadAccount.default_domain_root_account, type: :implicit, require_sis: false)
     sr_id = sis_pseudonym.ext_id
     school_id = sis_pseudonym.school_id
-    section_period_ids = quiz.course.course_sections.each.map { |section|
+    # section_period_ids = quiz.course.course_sections.each.map { |section|
+    section_period_ids = quiz.course.course_sections.each.select { |section|
+        puts "select " + section.inspect
+        section[:section_period_ids] != '' && section[:section_period_ids] != nil
```

Comment:
Not that it needs to change, but `!(section[:section_period_ids].empty? or section[:section_period_ids].nil?)` seems to be the Ruby way to go.

Reply:
I could go with `!(section[:section_period_ids].blank?)` 

From the docs: 

> blank?() public
> An object is blank if it’s false, empty, or a whitespace string. For example, false, ”, ‘ ’, nil, [], and {} are all blank.